_data/education.yml

```diff
@@ -253,6 +253,15 @@ websites:
       hardware: Yes
       doc: https://www.identity.psu.edu/services/authentication-services/two-factor/
 
+    - name: Purdue University
+      url: http://www.purdue.edu/
+      img: purdue.png
+      tfa: Yes
+      phone: Yes
```

Comment:
I looked through their docs site and I don't see mention of them supporting phone call. Is this accurate?

Reply:
I know duo allows the option to, but if their docs don't explicitly say it then it should be removed